test/product_factory.rb

```diff
@@ -36,6 +36,7 @@ def self.create_product(attrs = {})
     product["description"] = attrs["description"] || "SUSE Linux Enterprise #{product_name}."
     product["display_name"] = attrs["display_name"] || "SUSE Linux Enterprise #{product_name}"
     product["download_size"] = attrs["download_size"] || 0
+    # default: 2024-10-31
     product["eol"] = attrs["eol"] || 1730332800
```

Comment:
`eol` = end-of-life, the date after which the product is not supported anymore. You should see a warning about installing an unsupported product when you install it after that date.

This particular date is for SLE12-GM, IIRC.